src/BloomBrowserUI/bookEdit/toolbox/signLanguage/signLanguageTool.tsx

```diff
@@ -376,8 +376,10 @@ export class SignLanguageToolControls extends React.Component<
             // I found a couple of examples online with these rates for video/mp4 and the results
             // look reasonable. It's possible we could get useful recordings with lower rates.
             audioBitsPerSecond: 128000,
-            videoBitsPerSecond: 2500000,
-            mimeType: "video/mp4"
+            videoBitsPerSecond: 2500000
+            // Setting this after the move to Geckofx60 results in an error, but
+            // the default seems to produce the same result.
+            //mimeType: "video/mp4"
```

Comment:
I'm unsure about this... it's conceivable that the ff60 we're getting doesn't have the right codec, and we should raise the issue with @hindlemail